app/menu/main-menu.js

```diff
@@ -0,0 +1,76 @@
+const {Menu} = require('electron');
+const electron = require('electron');
+const app = electron.app;
+
+const template = [
+  {
+    label: 'Edit',
+    submenu: [
+      {
+        role: 'undo'
```

Comment:
Chrome does not have undo and redo in it's menu bar. I do not think that we should either.